build/flaws/unsafe-html.ts

```diff
@@ -80,6 +86,9 @@ function getAndMarkupUnsafeHTMLFlaws(doc, $, { rawContent, fileInfo }) {
   });
 
   $("*").each((i, element) => {
+    if (!isTag(element)) {
+      return;
+    }
```

Comment:
It's possible to avoid importing this function, and hence having to add another direct dependency in package.json with: `if (!("tagName" in element)) return;`